administrator-guides/file-upload/amazon-s3/README.md

```diff
@@ -21,23 +21,4 @@ You'll need to set the following _CORS Configuration_ to your bucket:
 
```

Comment:
@rodrigok the above is actually not applicable anymore now with UFS right?  Or maybe only if proxying is off?